hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/AvailabilityConditionEval.java

```diff
@@ -31,6 +31,8 @@
  */
 public class AvailabilityConditionEval extends ConditionEval {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
No need to remove them :-).
I remember that Eclipse tends to complain about it but I didn't see it on Idea, just curiosity.
It is fine to have them.